branchpro/tests/test_models.py

```diff
@@ -7,6 +7,7 @@
 
 import unittest
 import branchpro as bp
+import numpy as np
```

Comment:
The convention of grouping modules is in block of 

Imports of Python native packages, such as os, unittest etc

Imports of publicly distributed Python packages, such as numpy, scipy etc

Imports that are unique to this repo, e.g. branchpro. 

According to this convention could you please reorder the imports as 

Import unittest

Import numpy as np

Import branchpro as bp